realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/AppTests.kt

```diff
@@ -265,4 +267,25 @@ class AppTests {
         assertEquals(registry, app.getFunctions(user, registry).defaultCodecRegistry)
     }
 
+    @Test()
+    fun encryption() {
+        tearDown()
```

Comment:
We need it, only one App instance is allowed and setUp() is already creating one, if we don't remove we would not be able to create an App with an encryption key.

I don't like it neither, but because of we cannot control if setUp is applied or not (I didn't find a solution), we need to do this. 

We could although move the test case into another test class.

Reply:
Ah yeah. That make sense. We are setting one up in the `setUp` method already 👍